lib/types/library/LibraryBuilder.js

```diff
@@ -152,14 +152,18 @@ class LibraryBuilder extends AbstractBuilder {
 		}
 
 		this.addTask("buildThemes", async () => {
+			// Only compile themes directly below the lib namespace to be in sync with the theme support at runtime
+			// which only loads themes from that folder.
+			const inputPattern = `/resources/${project.metadata.namespace}/themes/*/library.source.less`;
```

Comment:
Apparently the namespace can be undefined here (see https://github.com/SAP/ui5-builder/pull/570/files#diff-60671e579ddf44d5c765da690b34a67cec4ed58ac1d0fb93bab7d1befb98578fR7). So a check and fallback is needed

Reply:
For `theme-library` I decided to rather stay with the existing pattern as we don't really know the namespaces. One solution would be to filter using the information of the available libraries, but that would be a bit more complex and requires that knowledge also within the `generateThemeDesignerResources` task.
I think for `theme-libraries` we can expect that it only contains `themes` folders that should be compiled.

Reply:
Ok, makes sense! However, we still to add a check for the namespace here, otherwise you might end up with `/resources/undefined/themes/*/library.source.less` here

Reply:
Done